src/pages/policies/[policyId].svelte

```diff
@@ -231,7 +226,13 @@ th {
 
   <div class="flex justify-between align-items-center">
     <h4>Items <span class="subtext">({approvedItems?.length} covered)</span></h4>
-    <Button on:click={toggleAllItems}>{showAllItems ? 'Active items...' : 'all items…'}</Button>
+    <div>
+      {#if isAboveTablet()}
+        <Checkbox label="Hide Inactive" on:checked={hideInactiveItems} on:unchecked={showInactiveItems} />
+      {:else}
+        <Switch on:selected={hideInactiveItems} on:deselected={showInactiveItems} label="Hide Inactive off/on" />
```

Comment:
I think we could probably remove "off/on" from the label:
```suggestion
        <Switch on:selected={hideInactiveItems} on:deselected={showInactiveItems} label="Hide Inactive" />
```